CHANGELOG.md

```diff
@@ -515,7 +515,35 @@ Other minor additions
   ```agda
   index-∈-lookup : (i : Fin n) (xs : Vec A n) → Any.index (∈-lookup i xs) ≡ i
   ```
-
+* Added new proofs to `Codata.Delay.Properties`:
+  ```agda
+  ⇓-unique : ∀ {a} → {A : Set a} →
+             {d : Delay A ∞} →
```

Comment:
In the CHANGELOG we typically drop the implicit arguments as
they are not interesting.